sdk/resourcemanager/elasticsans/armelasticsans/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 0.1.0 (2022-09-20)
+
+The package of `github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/elasticsans/armelasticsans` is using our [next generation design principles](https://azure.github.io/azure-sdk/general_introduction.html) since version 0.1.0, which contains breaking changes.
```

Comment:
Do we have track 1 SDK for elasticsans?  If not, shall we avoid mentioning breaking changes?  @xboxeer

Reply:
i can only see package github.com/Azure/azure-sdk-for-go/services/elastic/mgmt/2020-07-01 in track 1, not sure if they are refering to the sameting, need confirm with service team, is this pr raised on behalf of service team's ask?

Reply:
It should be a new RP. We use the same changelog content for any first version of track2. @xboxeer Could you help to provide a proper changelog content for the new onboard RP?